config/config.go

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
I'd expect to see a couple of unit tests for that new package. We want to easily verify through go test that a correctly formatted config file will work and a malformed one will fail as expected.